datanode/gateway/graphql/schema.graphql

```diff
@@ -4583,6 +4590,17 @@ type LiquiditySLAParameters {
   slaCompetitionFactor: String!
 }
 
+type LiquidationStrategy {
+  "Specifies the time in which the network will try to unload its position"
```

Comment:
```suggestion
  "Specifies the time at which the network will try to unload its position"
```

Reply:
Is this in a specific timestamp format?

Reply:
It's an int, number of seconds. It's not really _when_ something happens, but rather the period over which something happens. Combining all the parameters in this type, every block over the next N seconds, the network, if it has a non-zero position, will attempt to reduce its position by `total position * disposal_fraction` or `full_disposal_size` (if the position <= full disposal size) _or_ `order_book_volume * max_fraction_consumed`

So with a block time of 1s, a disposal time step of 20s, you could see the network place orders for the next 20 blocks.